### PR TITLE
Remove the magento tab from sale view

### PR DESCRIPTION
**Pull request by bhavana94**

### CODE REVIEW

**Comment by bhavana94** [comment]
https://3.basecamp.com/3194017/buckets/421793/todos/97766395#__recording_105765879

File: sale.xml [hunk 1]
```diff
@@ -24,6 +17,5 @@
             <field name="view" ref="sale.sale_view_form"/>
             <field name="act_window" ref="act_sale_form_all"/>
         </record>
-
     </data>
 </tryton>
```
Comment (riteshshrv) [inline sale.xml]: did you delete this file?? https://github.com/fulfilio/trytond-magento/blob/develop/view/sale_form.xml

Comment (bhavana94) [inline sale.xml]: Yes

Comment (riteshshrv) [inline sale.xml]: then add that deletion also to commit!

**Pull request closed without merging**